File: tests/go/cases/case_237_alias_hop_chain_04.go
package escape_tests

var retainedCase237 = []map[string]string{}

func Case237AliasHopChain04(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "alias_hop_chain_04",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "alias_hop_chain_04:" + raw,
	}
	a := payload
	b := a
	c := b
	// ESCAPE: multi-hop alias chain retained globally.
	retainedCase237 = append(retainedCase237, c)
	return "ok"
}

File: tests/nodejs/cases/case_237_alias_hop_chain_04.js
/**
 * Task case 237: alias_hop_chain_04 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case237AliasHopChain04(input) {
  const taskName = 'alias_hop_chain_04';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const a = payload;
  const b = a;
  const c = b;
  // ESCAPE: multi-hop alias chain into retained sink.
  retainedAudit.push(c);
  return 'ok';
}

module.exports = { case237AliasHopChain04 };
